pkg/network/node/ovscontroller.go

```diff
@@ -401,17 +424,29 @@ func (oc *ovsController) TearDownPod(hostVeth, podIP, sandboxID string) error {
 		}
 	}
 
-	err := oc.cleanupPodFlows(podIP)
+	if err := oc.cleanupPodFlows(podIP); err != nil {
+		return err
+	}
+
+	portUUIDs, err := oc.getPortsForSandbox(sandboxID)
```

Comment:
If we now absolutely depend on every port having a sandbox ID in its external-ids then we need to make AlreadySetUp() fail for bridges with non-external-id'ed ports attached. So change `ruleVersion` at the top of the file. (In which case you'll also want https://github.com/danwinship/origin/commit/dbbf89ac.)

UpdatePod() will also need to be updated to call ensureOvsPort().